package.json

```diff
@@ -20,7 +20,7 @@
     "test": "yarn test:unit:watch",
     "test:unit": "karma start ./test/unit/karma.local.conf.js",
     "test:unit:watch": "yarn test:unit --no-single-run",
-    "test:unit:bs": "node ./scripts/test/bs-wrapper.js karma start test/unit/karma.bs.conf.js",
+    "test:unit:bs": "node ./scripts/test/bs-wrapper.js karma start test/unit/karma.bs.conf.js && node ./scripts/test/bs-wrapper.js karma start test/unit/karma.bs.conf.js --ext",
```

Comment:
Question❓: shall we proceed with skipping the Browserstack tests for the extension completely as we were already running it on headless chrome?

Reply:
I think this would be simpler, yes!